src/renderer/styles/_button-state.scss

```diff
@@ -23,4 +23,10 @@
     background: $primary-light;
     cursor: default;
   }
+
+  &.disabled {
```

Comment:
I don't think we need to define a new class. If the element has `aira-disabled=true`, then we should be able to select it using something like:
```
&[aria-disabled="true"] {
    pointer-events: none;
    background: $primary-light;
    cursor: default;
}
```

Reply:
i did not know you could select like that :o